client/src/components/EditProfile/EditForm/EditForm.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from 'react';
+import TextField from '@material-ui/core/TextField';
+import useStyles from './useStyles';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+
+const EditForm = () => {
+  const classes = useStyles();
+
+  const [input, setInput] = useState({
+    firstName: '',
+    lastName: '',
+    gender: '',
+    birthday: '',
+    email: '',
+    phone: '',
+    address: '',
+    description: '',
+  });
+
+  const handleChange = (e: any) => {
+    const { name, value } = e.target;
+    setInput((oldState) => {
+      return { ...oldState, [name]: value };
+    });
+  };
+
+  const handleSubmit = (e: any) => {
+    e.preventDefault();
+    console.log('submitted');
+  };
+
+  return (
+    <div>
+      <Grid item xs>
+        <Typography className={classes.welcome} component="h1" variant="h5">
+          Edit Profile
+        </Typography>
+      </Grid>
+      <form onSubmit={handleSubmit} className={classes.form}>
+        <TextField
```

Comment:
You can possibly componentize this textfield as a lot of the attributes are repeated ones :)